src/matchers/toHaveBeenCalledOnce/predicate.js

```diff
@@ -0,0 +1,6 @@
+export default calls => {
```

Comment:
Looks great! I was curious as to why the false checks are explicit instead of something like
`export default calls => calls.length === 1`

Reply:
This is a prime example of 2/3rds TDD, I got some red, i made it green, i just didn't refactor 😆 
Illsort it in abit :P thanks for pointing out aha